docs/content/1.getting-started/2.migration.md

```diff
@@ -367,6 +367,59 @@ In addition to the renamed components, there are lots of changes to the componen
 This change affects the following components: `Breadcrumb`, `HorizontalNavigation`, `InputMenu`, `RadioGroup`, `Select`, `SelectMenu`, `VerticalNavigation`.
 ::
 
+2. `UNotifications` component is no longer available and needed due to the usage of `UApp`:
+
+```diff
+<template>
+- <UNotifications />
+</template>
+```
+
+3. `USlideover` now accepts `v-model:open` instead of `v-model`. Also, the content inside needs to be wrapped inside the `template #body`:
```

Comment:
This actually the case for many components like `Modal`, `Popover`, `DropdownMenu`, etc.

Reply:
Should I update this point then to also include these other components? Or remove this point in general? :)

Reply:
Yes you can do like the first point which is general and at the bottom a note about which components it's for

Reply:
Added :)